us.tylerc.cycles.yaml

```
@@ -0,0 +1,37 @@
+app-id: us.tylerc.cycles
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.golang
+command: cycles
+
+finish-args:
+    - --share=ipc 
+    - --socket=fallback-x11
+    - --device=dri 
+    - --socket=wayland
+
+    # Needed to support desktop notifications.
+    #- --talk-name=org.freedesktop.Notifications
+
+    - --filesystem=home
+
+build-options:
+  env:
+    - GOBIN=/app/bin
+    - GOROOT=/usr/lib/sdk/golang
+
+modules:
+    - name: cycles
+      buildsystem: simple
+      build-commands:
+        - $GOROOT/bin/go build -trimpath -o cycles
+        - install -Dm00755 cycles $FLATPAK_DEST/bin/cycles
+        - install -Dm00644 icon.png $FLATPAK_DEST/share/icons/hicolor/256x256/apps/$FLATPAK_ID.png
```

this is not the right size of the PNG.
```suggestion
        - install -Dm00644 icon.png $FLATPAK_DEST/share/icons/hicolor/512x512/apps/$FLATPAK_ID.png
```